sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -196,6 +196,7 @@ RUN apt-get update && apt-get install -y \
         libdb-dev \
         libiptc-dev \
         libxtables-dev \
+	libbpf-dev \
```

Comment:
Why is this needed?

Reply:
libbpf-dev is required for iproute2 version 5.10 (see it [here](https://salsa.debian.org/kernel-team/iproute2/-/blob/debian/5.10.0-4/debian/control#L17)).